### PR TITLE
Adding more niscope systemtests

### LINKED ISSUE #520: Write system tests for NI-SCOPE

### PR DESCRIPTION
**Pull request by injaleea**
[X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

~~[ ] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~~

~~[ ] I've added tests applicable for this pull request~~

### What does this Pull Request accomplish?

Adding more system tests for niscope.
test_configure_ref_levels
test_configure_trigger_digital
test_configure_trigger_edge
test_configure_trigger_hysteresis
test_configure_trigger_software
test_configure_trigger_video
test_configure_trigger_window
test_fetch_read_measuremet

### List issues fixed by this Pull Request below, if any.

Partial Fix #520 

### What testing has been done?
tox, system tests

### CODE REVIEW

File: src/niscope/system_tests/test_system_niscope.py [hunk 1]
```diff
@@ -125,3 +125,55 @@ def test_waveform_processing(session):
...
     session.horz_sample_rate == 10000000
+
+
+def test_configure_ref_levels(session):
+    session.configure_ref_levels()
+    session.commit()
+    assert session.meas_chan_high_ref_level == 90.0
+
+
+# revisit after #630 closed
```
Comment (marcoskirsch) [inline src/niscope/system_tests/test_system_niscope.py]: There's no way we'll ever find this. Do one of the following:

* Follow the convention and type this as TODO(lnjaleea): ... This is a string we search for.
* On the issue, make a note about revisiting this.

In this case, #630 is closed so just remove altogether.

File: src/niscope/system_tests/test_system_niscope.py [hunk 2]
```diff
@@ -125,3 +125,55 @@ def test_waveform_processing(session):
...
+def test_configure_trigger_digital(session):
+    session.configure_trigger_digital('VAL_RTSI_0')
+    session.vertical_range = 5
+    assert 'VAL_RTSI_0' == session.trigger_source
+
+
+def test_configure_trigger_edge(session):
+    assert niscope.TriggerSlope.POSITIVE == session.trigger_slope
+    session.configure_trigger_edge('0', niscope.TriggerCoupling.DC)
+    session.commit()
```
Comment (marcoskirsch) [inline src/niscope/system_tests/test_system_niscope.py]: Calling commit() is unnecessary in most (all?) of these tests. When you read an attribute, the driver will internally verify the session and compute all the coerced values for attributes. commit() is too heavy handed as it _also_ applies settings to hardware (not that it matters much in simulated devices).

Regardless, any redundant code is best removed.

Comment (injaleea) [reply src/niscope/system_tests/test_system_niscope.py]: Leaving a test with commit to check call to commit method. Removed redundant commit.

**Comment by marcoskirsch** [comment]
This closed #520 because of how the PR was written. Please be careful with this, it's happened multiple times. I reopened.

**Pull request merged**